src/lib/xmlparse.rs

```diff
@@ -84,6 +84,7 @@ pub use crate::lib::xmltok::*;
 pub use crate::stddef_h::{ptrdiff_t, size_t, NULL};
 use crate::stdlib::{memcmp, memcpy, memmove, memset};
 pub use ::libc::INT_MAX;
+use crate::lib::string_pool::StringPool;
```

Comment:
This could just be `crate::string_pool::StringPool`.

Reply:
Tried this, but it doesn't work with our current setup:
```Rust
error[E0432]: unresolved import `crate::string_pool`
  --> src/lib/xmlparse.rs:66:12
   |
66 | use crate::string_pool::StringPool;
   |            ^^^^^^^^^^^
   |            |
   |            unresolved import
   |            help: a similar path exists: `crate::lib::string_pool`
```

Reply:
Yeah, everything is nested inside of `lib`. We should fix that at some point.